Include/refcount.h

```diff
@@ -396,6 +440,24 @@ static inline void Py_DECREF(const char *filename, int lineno, PyObject *op)
 #define Py_DECREF(op) Py_DECREF(__FILE__, __LINE__, _PyObject_CAST(op))
 
 #else
+static inline void Py_DECREF_MORTAL(PyObject *op)
+{
+    _Py_DECREF_STAT_INC();
```

Comment:
please add an assert here that it's not immortal.

Reply:
It might be immortal, if the object was made immortal after the reference was created.
We can assert that it isn't *static* immortal, which is almost as good.